src/providers/twitch/api/Helix.hpp

```diff
@@ -340,6 +340,19 @@ enum class HelixDeleteChatMessagesError {
     Forwarded,
 };
 
+enum class HelixAddChannelModeratorError {
+    Unknown,
+    UserMissingScope,
+    UserNotAuthenticated,
+    UserNotAuthorized,
```

Comment:
Unused error code

Reply:
I reused this for the `incorrect user authorization` error, as the enum name seemed to make the most sense 👍